### PR TITLE
fix(card): adjust story layouts to align left with top spacing

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

#4399 

### Description

Updated Card and Card Link to align left and have top padding of 1 rem, so that react and web components are aligned

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: packages/web-components/src/components/card/__stories__/card.stories.ts [hunk 1]
```diff
@@ -101,9 +101,9 @@ export default {
   title: 'Components/Card',
   decorators: [
     story => html`
-      <div class="dds-ce-demo-devenv--simple-grid dds-ce-demo-devenv--simple-grid--card">
-        <div class="bx--row dds-ce-demo-devenv--grid-row">
-          <div class="bx--col-sm-4 bx--col-md-3 bx--col-lg-12 bx--col-xlg-8 bx--no-gutter">
+      <div class="bx--grid dds-ce-demo-devenv--grid--stretch">
+        <div class="bx--row">
+          <div class="bx--col-sm-4 bx--col-md-3 bx--col-lg-6 bx--col-xlg-4 bx--no-gutter">
```
Comment (asudoh) [inline packages/web-components/src/components/card/__stories__/card.stories.ts:106]: Actually the codebase is transitioning away from `bx--grid` class as part of the effort described in #4174 epic. That said, would you try updating the CSS rulesets for `.dde-ce-demo-devenv--simple-grid--card`? You can find them at: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.15.0-rc.2/packages/web-components/.storybook/container.scss#L77-L101

Similar thing applies to card link story. Thanks!

**Review by asudoh — COMMENTED** [review]
Thank you for the update working with the designers @ariellalgilmore!

**Comment by asudoh** [comment]
@ariellalgilmore Thank you for continuing working on this! I see some non-related change now... Probably a wrong rebase?

**Comment by ariellalgilmore** [comment]
closing and creating new PR

**Pull request closed without merging**